Pages/FeedbackPage/FeedbackLogged.cshtml.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FoodKept.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+
+namespace FoodKept.Pages.FeedbackPage
+{
+    public class FeedbackLoggedModel : PageModel
+    {
+        private readonly FoodKept.Data.ShopContext _context;
+        private readonly UserManager<ApplicationUser> _userManager;
+        public FeedbackLoggedModel(FoodKept.Data.ShopContext context, UserManager<ApplicationUser> userManager)
+        {
+            _context = context;
+            _userManager = userManager;
+        }
+        [BindProperty]
+        public Feedback Feedback { get; set; }
+        public ApplicationUser ApplicationUser { get; set; }
```

Comment:
Is this needed? (ApplicationUser)

Reply:
At this moment not really but in the future it could be.